demisto_sdk/commands/validate/validators/BA_validators/BA127_is_valid_context_path_depth.py

```diff
@@ -0,0 +1,115 @@
+                    invalid_paths = self.is_context_depth_less_or_equal_to_5_command(command, old_command)
+                    if invalid_paths:
+                        results.append(
+                            ValidationResult(
+                                validator=self,
+                                message=self.error_message.format(
+                                    command.name, invalid_paths
+                                ),
+                                content_object=content_item,
+                            )
+                    )
+        return results
+
+    def is_context_depth_less_or_equal_to_5_command(self, command: Command, old_command: Command | str):
+        """Validate that all outputs entry has contextPath key for a given command.
+
+        Args:
+            command (Command): The command to run on
+            old_command (Command | str): The old command, to compare if the output is new or already existed
+            if the command is new, then it will be empty
+
+        Returns:
+           List of bad context paths if the contextPath depths is bigger then 5. Otherwise, return False.
+        """
+        wrong_depth_values = []
+        for output in command.outputs:
+            if old_command:
+                if output in old_command.outputs:
+                    continue
+            output_depth = len(output.contextPath.split('.'))
+            if output_depth > 5:
+                wrong_depth_values.append(output.contextPath)
+        if wrong_depth_values:
+            wrong_values_string = '\n'.join(wrong_depth_values)
+            return wrong_values_string
+        else:
+            return False
+
+
+    def is_context_depth_less_or_equal_to_5_script(self, content_item: Iterable[ContentTypes],
+                                                   old_content_item: Iterable[ContentTypes] | str ):
+        """Validate that all outputs entry has contextPath key for a given command.
+
+        Args:
+            content_item (Iterable[ContentTypes]: The content item to run on
+            old_content_item (Iterable[ContentTypes] | str): The old content item, to compare if the output is new or
+            already existed. If the content item is new, then it will be empty
+        Returns:
+             List of bad context paths if the contextPath depths is bigger then 5. Otherwise, return False.
+        """
+        wrong_depth_values = []
+        for output in content_item.outputs:
+            if old_content_item:
+                old_ouptuts = old_content_item.outputs
+                if output in old_ouptuts:
+                    continue
+            output_depth = len(output.contextPath.split('.'))
+            if output_depth > 5:
+                wrong_depth_values.append(output.contextPath)
+        if wrong_depth_values:
+            wrong_values_string = '\n'.join(wrong_depth_values)
+            return wrong_values_string
+        else:
+            return False
```

Comment:
you can return it as a list right away, if the list is empty then it count as False.